code-challenges.js

```diff
@@ -26,24 +56,69 @@ const temp3 = 212
+//
+var mergeCode = myNumbers1.concat(myNumbers2)
+console.log(mergeCode.length)
 
+//so i will use a built in method .concat two merge num1 and num2
+//then I would call on the code with the .length to get the full length between two codes now that they are merged.
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "1202 ohcE"
-
 const currentCohort = "Alpha 2022"
+var reverseMe = currentCohort.split("")
+console.log(reverseMe.reverse())
+//create a code with a built in method that would reverse the string
+//input the variable name with the built in code in order to get results
+//output reverse string
 
-
+//So i tried to go with the origin plan. Of logging the variable name and adding the built in method .reverse. But it didn't work because it's one string. Not even a collection of list. One way to over cone that is if i were to split every single little letter into it's own string and then use the reverse method on it.
 // --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
-
+//
 const myArray = [13, 34, 5, 10, 27, 42]
+for (let i = 0; i < myArray.length; i++){
+  if(myArray[i] % 2 === 0){
+    console.log("even")
+  } else {
+    console.log("odd")
+  }
+}
+
+// console.log(myArray)
+
+//create a statement that is called evenOrOdd
+//input. would use the for loops that way i am able to go through each element inside the array
+//starting point= 0. ending point has to be length of the myarray. then it will iterate up. or in increments
+//once i create the for loop i will incorporate the modulas as part as the conditional statement. to evaluate whether each value is oven or odd.
+
+//output: will output a list of strings that has a variation of even and odd
+//if statement is true (even) then return string 'even'. else return 'odd' if false
 
 
 // --------------------5) Create a function that takes in a set of any two numbers and subtracts the smaller number from the larger number using the sets of test variables provided below. Expected output: 42, 3
 
+//create a function called subtraction
+//input
+//will take in two parameters: num1 and num2
+//use conditional to see what number is the smallest
+//if num1 is the smallest then it will subtract from num2
+//if not then it would return num2-num1
+
+//output
+//return a number that is subtracts the biggest with the smallest number
+
+const subtraction = (num1, num2) => {
+  if (num1 < num2){
+    return (num2 - num1)
+  } else {
+    return (num1 - num2)
+  }
+}
+// console.log(subtraction(num1,num2))
 // Set one:
 const number1 = 58
 const number2 = 100
-
+//with this one I was hoping to pass the variable through the parameters when I called on the function but it was giving me an error. I got the answer right but wasn't as satisfying.
```

Comment:
Your function looks perfect, the error is from declaring the variable after the console log.  Try to set variables before the function when possible.  Javascript runs from top to bottom, so if a function is called with a variable that has not been created yet, it will say the variable does not exist.